r/R/marker_heatmap.R

```diff
@@ -9,13 +9,21 @@
 #' @examples
 runMarkerHeatmap <- function(req, data) {
   nFeatures <- req$body$nGenes
-  data <- getClusters(req$body$type, req$body$config$resolution, data)
+  cellSets <- req$body$cellSets$children
+  data$custom <- NA
 
-  all_markers <- presto::wilcoxauc(data, assay = "data", seurat_assay = "RNA")
+  object_ids <- data$cells_id
+  for (i in 1:length(cellSets)){
```

Comment:
```suggestion
  for (i in seq_along(cellSets)) {
```

Less error prone as it will skip iteration is cellSets in length zero